src/components/JoinUs/JoinUs.js

```diff
@@ -25,7 +25,7 @@ class JoinUs extends Component {
 
   validateField(value) {
     const emailRegEx = /^(\D)+(\w)*((\.(\w)+)?)+@(\D)+(\w)*((\.(\D)+(\w)*)+)?(\.)[a-z]{2,}$/;
-    const fieldValidationError = this.state.formError;
+    const { fieldValidationError } = this.state.formError;
```

Comment:
it should be
`const { formError, emailValid } = this.state;`
or
`const { formError: fieldValidationError, emailValid } = this.state;`